UnityProject/Assets/Scripts/Items/Wieldable.cs

```diff
@@ -0,0 +1,162 @@
+using Messages.Server;
+using Mirror;
+using UnityEngine;
+
+namespace Items
+{
+	public class Wieldable : NetworkBehaviour, IServerInventoryMove, ICheckedInteractable<HandActivate>
+	{
+		[SerializeField]
+		private int damageUnwielded;
+
+		[SerializeField]
+		private int damageWielded;
+
+		public ItemsSprites Wielded = new ItemsSprites();
+		public ItemsSprites Unwielded = new ItemsSprites();
+
+		[SyncVar(hook = nameof(SyncState))]
+		private bool isWielded;
+
+		private ItemAttributesV2 itemAttributes;
+
+		public enum hiddenHandValues
+		{
+			bothHands = 0,
+			leftHand = 1,
+			rightHand = 2,
+			none = 3
+		}
+
+		private void Awake()
+		{
+			itemAttributes = GetComponent<ItemAttributesV2>();
+		}
+
+		private void OnEnable()
+		{
+			HandsController.OnSwapHand.AddListener(OnSwapHands);
+		}
+
+		private void OnDisable()
+		{
+			HandsController.OnSwapHand.RemoveListener(OnSwapHands);
+		}
+
+		private void OnSwapHands()
+		{
+			if (isWielded)
+			{
+				Chat.AddExamineMsgFromServer(PlayerManager.LocalPlayerScript.gameObject, $"Your other hand is too busy holding {gameObject.ExpensiveName()}!");
+				HandsController.OnSwapHand.RemoveListener(OnSwapHands);
+				HandsController.Instance.SwapHand();
```

Comment:
```suggestion
				HandsController.SwapHand();
```

Ah sorry, didnt see that SwapHand was static

Reply:
Ill commit everything you propose